client/src/thunks.ts

```diff
@@ -6,16 +7,16 @@ import { AppState } from "./store";
 import { addBit, changeVote, replaceBits } from './store/bits/actions';
 import { changeSort, setLabels, setMainCharacters, setPageSize, setStages, setVsCharacters } from "./store/filtering/actions";
 import { setProfile } from "./store/profile/actions";
-import { Bit, CharacterId, LabelId, PageSize, SortOption, StageId, Vote } from "./types";
+import { Bit, Character, Label, PageSize, SortOption, Stage, Vote } from "./types";
 import { buildUriFromState } from "./uri_util";
 
 type AppThunkAction = ThunkAction<Promise<void>, AppState, null, AnyAction>
 type AppThunkActionCreator = ActionCreator<AppThunkAction>
 
 export const thunkFetchBits: AppThunkActionCreator = () => {
   return async (dispatch, getState) => {
-    const response = await fetchBitsApi(getState().filtering)
-    dispatch(replaceBits(response.bits));
+    const bits: Bit[] = await fetchBitsApi(getState().filtering);
```

Comment:
This should be inferred. Consider adding a return type to fetchBitsApi if it's ambiguous.

Reply:
Done.